AlastriaID/VerifiableAttestation/2022-11/README.md

```diff
@@ -0,0 +1,422 @@
+    </td>
+  </tr>
+
+
+  <tr>
+    <td>credentialSchema</td>
+    <td>
+      Contains information about the credential schema (template) on which the Verifiable Attestation is based (via the Trusted Schemas Registry, TSR).<br>
+      Definition: <a href="https://www.w3.org/TR/vc-data-model/#data-schemas">https://www.w3.org/TR/vc-data-model/#data-schemas</a> 
+    </td>
+    <td>object</td>
+    <td>
+    </td>
+  </tr>
+
+
+  <tr>
+    <td>evidence</td>
+    <td>
+      Contains information about the process which resulted in the issuance of the Verifiable Attestation.<br><br>
+      Definition: <a href="https://www.w3.org/TR/vc-data-model/#evidence">https://www.w3.org/TR/vc-data-model/#evidence</a> 
+    </td>
+    <td>array&ltobject&gt</td>
+    <td>
+      Evidence can contain one or more evidence objects.
+    </td>
+  </tr>
+
+  <tr>
+    <td>proof</td>
+    <td>
+      Contains information about the proof.<br><br>
+      Definition: <a href="https://www.w3.org/TR/vc-data-model/#proofs-signatures">https://www.w3.org/TR/vc-data-model/#proofs-signatures</a> 
+    </td>
+    <td>object</td>
+    <td>
+      EBSI foresees the possibility to use different types of proofs for Verifiable Credentials, such as proofs derived from eIDAS keys (qualified) to DID keys (unqualified).  In EBSI 2.0, every Verifiable Attestation will only contain a single proof, which must be derived from eIDAS keys.<br><br>
+      The proof property must only be present when Linked Data Signatures are used (which are currently unsupported)<br><br>
+      For more information about proofs, check: E-signing and e-sealing Verifiable Credentials and Verifiable Presentations<br><br>
+      This proof field is optional because the proof can be external instead of internal. 
+    </td>
+  </tr>
+
+
+  <tr>
+    <td><s>termsOfUse</s></td>
+    <td>
+      Defines the terms under which the Verifiable Attestation was issued.<br><br>
+      Definition: <a href="https://www.w3.org/TR/vc-data-model/#terms-of-use">https://www.w3.org/TR/vc-data-model/#terms-of-use</a> 
+    </td>
+    <td>array&ltobject&gt</td>
+    <td>
+      <s>E.g., terms of use can contain information about the accreditation of the issuer of the Verifiable Attestation. Terms of use can contain one or more references to accreditations. </s> 
+    </td>
+  </tr>
+</table>
+
+<br></br>
+
+## Credential subject
+All credential subject objects MUST contain property id. Other properties are defined in data models of use cases.
+
+<table>
+  <tr>
+    <th>Terminology</th>
+    <th>Description</th>
+    <th>Type</th>
+    <th>Notes</th>
+  </tr>
+
+  <tr>
+    <td>id</td>
+    <td>
+        Defines the DID of the subject that is described by the Verifiable Attestation.<br><br>
+        Definition: <a href="https://www.w3.org/TR/vc-data-model/#identifier">https://www.w3.org/TR/vc-data-model/#identifier</a> 
+    </td>
+    <td>string</td>
+    <td>
+    </td>
+  </tr>
+</table>
+
+<br></br>
+
+## Credential status
```

Comment:
CredentialStatus should be crossed out down here too, shouldn't it?

Reply:
This can't be done because markdown doesn't let me strike that title but the table below is crossed out to show that we will not be implementing it in that way.